app/views/controllers/shared/_search_form.erb

```diff
@@ -2,10 +2,19 @@
 <%
 form_args = { url: { action: :create }, model: search,
               id: "#{controller.search_type}_search_form",
-              class: "faceted-search-form" }
-# form_args[:model] = search if search&:id
+              class: "faceted-search-form pb-4" }
+toggle_classes = %w[btn btn-link navbar-link px-2]
```

Comment:
Should this be frozen or a constant? No answer required.

Reply:
Thanks - actually it doesn't need to be there at all. I moved the toggles to helpers.